idea-plugin/src/main/kotlin/arrow/meta/ide/plugins/purity/PurityPlugin.kt

```diff
@@ -24,14 +24,16 @@ val IdeMetaPlugin.purity: IdePlugin
       addLocalInspection(
         inspection = purityInspection,
         level = HighlightDisplayLevel.ERROR,
-        groupPath = ArrowPath + "PurityPlugin"
+        groupPath = ArrowPath + "PurityPlugin",
```

Comment:
Shouldn't we add these also as string resources? The same way you did for the rest of the paths, and just add the English one if that makes sense (since we don't want to translate them).